vortex-array/src/array/varbinview/compute.rs

```diff
@@ -63,10 +63,10 @@ impl SliceFn for VarBinViewArray {
 /// Take involves creating a new array that references the old array, just with the given set of views.
 impl TakeFn for VarBinViewArray {
     fn take(&self, indices: &Array) -> VortexResult<Array> {
-        let array_arrow = self.clone().into_array().into_canonical()?.into_arrow()?;
+        let array_ref = varbinview_as_arrow(self);
```

Comment:
this one line made a ~20x difference in take performance.

take_strings benchmark was 40µs with old version, mostly due to the `try_from`.

This brings it down to 2µs